components/PostCardList/PostCardListItem.tsx

```diff
@@ -0,0 +1,213 @@
+import * as React from "react";
+import styled from "styled-components";
+import { MOBILE_MEDIA } from "../../constants/mediaquery";
+import Post from "../../models/Post";
+import calculateFinalPot from "../../utilities/calculateFinalPot";
+import getPositionByPlayerAndIndex from "../../utilities/getPositionByPlayerAndIndex";
+import getRelativeDateString from "../../utilities/getRelativeDateString";
+import getSIMetricPrefixData from "../../utilities/getSIMetricPrefixData";
+import Card from "../Card";
+import { ThumbsUpIcon } from "../Icon";
+import PlayingCard from "../PlayingCard";
+import PostTypeContext, { PostType } from "./PostTypeContext";
+
+interface Props extends React.Attributes {
+  post: Post;
+  onClick?: React.MouseEventHandler<HTMLElement>;
+  className?: string;
+  style?: React.CSSProperties;
+}
+
+export default function PostCardListItem({ post, ...props }: Props) {
+  const { heroIndex, playerCards } = post.gameSituation;
+  const { left, right } = playerCards[heroIndex]!;
+  const postType = React.useContext(PostTypeContext);
+  const [date, mobileDateUnit, dateUnit] = getRelativeDateString(
+    postType === PostType.recent ? post.createdAt : post.lastUpdatedAt
+  );
```

Comment:
It would be better if you split it into two functions, such like `getRelativeDateString()` and `getRelativeShortDateString`